src/main/java/com/mealmentor/enterprise/service/MealPlanServiceStub.java

```diff
@@ -27,7 +27,7 @@ public MealItem fetchById(int id) {
     }
 
     @Override
-    public void delete(int id) throws Exception {
```

Comment:
I took out an unused throw Exception as it wasn't needed within the code.
Source: https://www.baeldung.com/java-clean-code

Reply:
Is it in the interface?